test/test_Autocommands.py

```diff
@@ -0,0 +1,31 @@
+# encoding: utf-8
+from test.vim_test_case import VimTestCase as _VimTest
+from test.constant import *
+
+# Autocommands  {{{#
+
+class Autocommands(_VimTest):
```

Comment:
I think you want a more complex test too - a snippet with more than $0 so that you can expand a recursive snippet in it too.